tests/Unit/Evolution/Systems/GeneralizedHarmonic/Test_Constraints.cpp

```diff
@@ -538,6 +538,138 @@ void test_constraint_energy_random(const DataType& used_for_size) noexcept {
                  // failures of this test, suspected from
                  // accumulated roundoff error
 }
+
+// Test the return-by-reference four-index constraint
```

Comment:
This comment needs to be updated from `four-index constraint` -> `constraint energy`